USE/IndicShapingInvalidCluster.txt

```diff
@@ -8,98 +8,106 @@
 #
 # Scope: This file enumerates sequences of characters that should be treated as invalid clusters
 
-  0905 0946       ; # DEVANAGARI LETTER A, DEVANAGARI VOWEL SIGN SHORT E

```

Comment:
This is mostly whitespace changes to allow width for Khojki sequence with three code points.